src/Util_TEST.cc

```diff
@@ -1032,3 +1032,83 @@ TEST_F(UtilTest, LoadMesh)
   EXPECT_NE(nullptr, optimizedMesh);
   EXPECT_EQ(16u, optimizedMesh->SubMeshCount());
 }
+
+/////////////////////////////////////////////////
+TEST_F(UtilTest, MeshAxisAlignedBoundingBox)
+{
+  sdf::Mesh meshSdf;
+  math::AxisAlignedBox emptyBox, aab;
+  EXPECT_EQ(emptyBox, meshAxisAlignedBox(meshSdf));
+
+  meshSdf.SetUri("invalid_uri");
+  meshSdf.SetFilePath("invalid_filepath");
+  EXPECT_EQ(emptyBox, meshAxisAlignedBox(meshSdf));
+
+  meshSdf.SetUri("name://unit_box");
+  aab = meshAxisAlignedBox(meshSdf);
+  EXPECT_NE(emptyBox, aab);
+  EXPECT_EQ(aab.Size(), math::Vector3d::One);
+  EXPECT_EQ(aab.Min(), math::Vector3d(-0.5, -0.5, -0.5));
+  EXPECT_EQ(aab.Max(), math::Vector3d(0.5, 0.5, 0.5));
+  EXPECT_EQ(aab.Center(), math::Vector3d::Zero);
+
+  meshSdf.SetUri("duck.dae");
+  std::string filePath = common::joinPaths(std::string(PROJECT_SOURCE_PATH),
```

Comment:
include `<string>`

Reply:
Done in https://github.com/gazebosim/gz-sim/pull/2787/commits/469a44634d25c941a6bec80c4f5652c8ec160a9f